install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1370,6 +1370,26 @@ of Fluentd pods, the deployer does delete the *logging-fluentd* Daemonset and re
 it from the *logging-fluentd-template* template.
 ====
 
+The latest EFK stack now uses Elasticsearch 2.3 with a common data model. This
+means Fluentd sends logs to Elasticsearch with a new indexing pattern for
+projects. The pattern is:
++
+----
+project.{namespace_name}.{namespace_id}.YYYY.MM.DD
+----
++
+For example:
++
+----
+project.logging.5dad9bd0-a7a1-11e6-94a0-5254000db84b.2016.11.14
+----
+
+[IMPORTANT]
+====
+Downgrading from Elasticsearch 2.3 to Elasticsearch 1.x is not possible due to
+migration to the new data model.
```

Comment:
Technically this is independent of the common data model; this is simply a limitation of Elasticsearch 1.x not being able to interpret the data structure of ES 2.x.

It may not be necessary to capture that in the documentation, however it should be noted.